### ISSUE TITLE
Enable interactive account creation

### ISSUE THREAD

**Issue by kenperkins** [jrperritt/rack#188]
The only case I'd *love* to have for public beta is provisioning a new account, ideally with an interactive prompt.

i.e.

```
$ rack register
Register a new Rackspace Cloud account:
Username: _
```
Username:

**Comment by jrperritt** [comment]
For now, I'm marking this as a question and assigning it to @kenperkins . We need to find out which API to query for this, and how it differs, if at all, from the signup process on rackspace.com. In addition, we should consider any vulnerabilities this feature may allow.

**Comment by jrperritt** [comment]
If we can address the questions/issues above :point_up: this week, we can have this for `public beta`.

**Comment by rgbkrk** [comment]
Yup, I'm looking forward to this. Any word on the API @kenperkins?

**Comment by jrperritt** [comment]
@kenperkins was this determined as doable or not?
